tests/rules/javascript/test.no_eval.js

```diff
@@ -0,0 +1,114 @@
+import { VALIDATION_WARNING } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+
+// These rules were mostly copied and adapted from
+// https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-eval.js
+// Please make sure to keep them up-to-date and report upstream errors.
+describe('no_eval', () => {
+  var validCodes = [
+    'Eval(foo)',
```

Comment:
This misses the cases in the src test file that this shouldn't raise an issue for setTimeout etc or is that a problem in this case due to the other rules we have?